src/java/picard/cmdline/CommandLineParser.java

```diff
@@ -363,7 +395,7 @@ public void htmlUsage(final PrintStream stream, final String programName, final
         stream.println("<a id=\"" + programName + "\"/>");
         stream.println("<h3>" + programName + "</h3>");
         stream.println("<section>");
-        stream.println("<p>" + getUsagePreamble() + "</p>");
+        stream.println("<p>" + htmlEscape(getUsagePreamble()) + "</p>");
```

Comment:
Actually I meant I should have thought of the unescape the unescape for regular usage. Why would you want to strip out html elements in the html usage section???